frontend/src/components/EmployeeModal/EmployeeModal.tsx

```diff
@@ -53,12 +53,13 @@ const EmployeeModal = ({ existingEmployee }: EmployeeModalProps) => {
   );
   const [imageBase64, setImageBase64] = useState('');
   const { withDefaults } = useReq();
-  const { refreshAdmins, refreshDrivers } = useEmployees();
+  const { refreshAdmins, refreshDrivers, drivers } = useEmployees();
```

Comment:
What do you use drivers for? It doesnt seem like it is used in employeeModal at all.